Dispatcher/Repositories/ArticlesRepository.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ArticlesRepository.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+class ArticlesRepository {
+
+    let alamofireManager = AlamofireManager()
+    var articles: [Article] = []
+    
+    func getData(completionHandler: @escaping (_ articles: [Article], _ errorMsg: String?) -> Void) {
+        let request = Request(baseUrl: APIConstants.newscatcherURL, headers: ["x-api-key": APIConstants.APIkey], parameters: ["countries" : "IL"], method: .get)
```

Comment:
Since we'll have the same headers, parameters and method - we wouldn't want to write the same initialization of Request over and over again for every request that we will want to send out.